app/static/js/index.js

```diff
@@ -172,9 +175,26 @@ function runSearch() {
     p.list.innerHTML = '';
     pagesLoaded = 0;
     pagesFinished = false;
+    window.history.pushState(nextState, "Updated URL Search Query", createUrl(criteria));
     loadNextPage();
 }
 
+function createUrl(criteria) {
```

Comment:
I think there's no need to take `criteria` as a parameter here since it's global in the file